app/models/lesson.rb

```diff
@@ -0,0 +1,3 @@
+class Lesson < ApplicationRecord
+  has_many :answer
```

Comment:
if you want to use `has_many` relationship, the model should be in plural form. Please change this to `has_many :answers`